frontend/src/components/sightingCard/sightingCard.tsx

```diff
@@ -12,28 +19,70 @@ export const SightingCard: React.FunctionComponent<SightingCardProps> = ({
   const species = sighting.species;
   const formattedDate = format(parseISO(sighting.date), "do MMMM, yyyy");
 
+  let descriptionSection = <></>;
+  if ("description" in sighting) {
+    descriptionSection = <p>Description: {sighting.description}</p>;
+  }
+
+  let locationSection = <></>;
+  if ("location" in sighting) {
+    locationSection = (
+      <>
+        <p>Latitude: {sighting.location.latitude}</p>
+        <p>Longitude: {sighting.location.longitude}</p>
+      </>
+    );
+  } else {
+    locationSection = (
+      <>
+        <p>Latitude: {sighting.latitude}</p>
+        <p>Longitude: {sighting.longitude}</p>
+      </>
+    );
+  }
+
+  let speciesSection = <></>;
+  if (Array.isArray(species)) {
+    console.log(species.map);
+    speciesSection = (
+      <>
+        {species.map((s) => {
+          return (
+            <>
+              <p>Species Name: {s.name}</p>
+              <p>Species Description: {s.description}</p>
+            </>
+          );
+        })}
+      </>
```

Comment:
I think this should be in some sort of `<ul>`, to give it a more semantic structure